tests/app/job/test_rest.py

```diff
@@ -715,3 +715,31 @@ def create_10_jobs(db, session, service, template):
         for _ in range(10):
             the_time.tick(timedelta(hours=1))
             create_job(db, session, service, template)
+
+
+def test_get_all_notifications_for_job_returns_csv_format(
+    client,
+    notify_db,
+    notify_db_session,
+):
+    job = create_job(notify_db, notify_db_session)
+    notification = create_notification(
+        notify_db,
+        notify_db_session,
+        job=job,
+        job_row_number=1,
+        created_at=datetime.utcnow(),
+    )
+
+    response = client.get(
+        path='/service/{}/job/{}/notifications?'.format(notification.service.id, job.id, True),
```

Comment:
We follow this pattern across the codebase. I think it's okay here, key thing is `query_string` clearly identifies the important parameter.